### DISCUSSION TITLE
Problem with Chrome Version 123.0.6312.59

### DISCUSSION THREAD

**Discussion by c4nier** [humanwhocodes/humanfs#108] [Q&A]
I tried to use humanfs for the web in a React+Vite application but i got this error:
X [ERROR] Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 2 overrides). Is there something to do to fix that error? Thank you.

**Comment by nzakas** [comment]
This error is coming from Vite, so you'd need to ask them for help. 

`@humanfs/web` does use top-level await, so you'll need to use it in an environment that supports it.

**Reply by c4nier** [reply]
Thanks for your response
